components/WarningPage.js

```diff
@@ -0,0 +1,11 @@
+import Link from 'next/link';
+
+export default function WarningPage() {
+  return (
+    <section>
+      <Link href="/warning">
+        <h3>!</h3>
```

Comment:
Maybe you could use a more descriptive headline. That way you would have a larger space to click on and it's better for the user experience ;)